app/policies/grant_submission/submission_policy.rb

```diff
@@ -39,7 +39,7 @@ def show?
   end
 
   def create?
-    user.system_admin? || (user.present? && @grant.accepting_submissions?)
+    (user.present? && @grant.accepting_submissions?) || grant_editor_access? || user.system_admin?
```

Comment:
is there a test for allowing grant editor to create submission when it's not accepting submission?